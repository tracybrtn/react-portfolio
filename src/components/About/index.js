import React from "react"

//About me section is the default section
import avatarImage from "../../../src/assets/images/Avatar-Tracy.png";

function About() {
  return(
    <section id="about-me" className="d-flex flex-column">
      <div className="box">
        <div>
          <h2>About me</h2>
        </div>
        <div className="container">
          <img src={avatarImage} alt="Tracy Burton Bravo smiling."></img>
        </div>
        <div className="description">
          <p>
          Full stack web developer and global affairs graduate with an extensive record of teamwork and leadership experience. Motivated and reliable professional who thrives in fast-paced, dynamic environments. Third-culture kid, bilingual English and Spanish speaker, pop-culture connoisseur.
          </p>
        </div>
      </div>
    </section>
  );
}

export default About;