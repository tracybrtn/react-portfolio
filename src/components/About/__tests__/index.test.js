//import dependencies
import React from "react";
import { render, cleanup } from '@testing-library/react';
import About from '..';

//cleanup function
afterEach(cleanup);

describe('About component', () => {
  it('renders', () => {
    render(<About />);
  });
  it('matches snapshot DOM node structure', () => {
    //asFragment returns a snapshot of the About component
    const { asFragment } = render(<About />);
    expect(asFragment()).toMatchSnapshot();
  });
})