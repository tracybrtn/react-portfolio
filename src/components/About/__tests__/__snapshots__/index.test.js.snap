// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="d-flex"
    id="about-me"
  >
    <h2>
      Hi! It's Tracy.
    </h2>
    <p
      class="description"
    >
      I am a 
      <span>
        MERN stack developer
      </span>
       and global affairs graduate with an extensive record of teamwork and leadership experience.
    </p>
    <p
      class="description"
    >
      Motivated and reliable professional who thrives in 
      <span>
        fast-paced and dynamic environments
      </span>
      .
    </p>
    <p
      class="description"
    >
      Third-culture kid, 
      <span>
        bilingual English and Spanish speaker
      </span>
      , pop-culture connoisseur, and family genealogist.
    </p>
    <div
      class="avatar-container"
    >
      <img
        alt="Tracy Burton Bravo smiling."
        src="Avatar-Tracy.png"
      />
    </div>
    <h3>
      Welcome to my portfolio!
    </h3>
  </section>
</DocumentFragment>
`;
